### DISCUSSION TITLE
0.9.3.7 - problem editing procedures

### DISCUSSION THREAD

**Discussion by steveh454** [mariuz/flamerobin#186] [General]
I downloaded latest version and could not edit any stored procedures. 
Error appears: -206 Column does not belong to referenced table. 
Version 0.9.2.1851 works fine on the same database. 
I have the 2 versions of FlameRobin installed. It is a great tool but pity that the latest version has this problem.

**Comment by luronumen** [comment]
Hi @steveh454,

I retested using the  [0.9.3.7 Snapshot](https://github.com/mariuz/flamerobin/releases/tag/0.9.3.7) and I was not able to reproduce the issue mentioned by you.

Could you please confirm the steps to reproduce the issue?
Right mouse click on a stored procedure and then select Alter.

**Comment by Jdochoa** [comment]
Hi @steveh454 
The error is you version of firebird. Please confirm if you used FB 2.1 or minor.
I made the mistake of not validating the firebird version
It will be corrected in the next snapshot.

./jo
